frontend/packages/kubevirt-plugin/src/components/vms/vm.tsx

```diff
@@ -62,42 +91,103 @@ const VMHeader = () =>
   );
 
 const VMRow: React.FC<VMRowProps> = ({
-  obj: vm,
+  obj,
   customData: { pods, migrations, vmiLookup, migrationLookup },
   index,
   key,
   style,
 }) => {
   const dimensify = dimensifyRow(tableColumnClasses);
-  const name = getName(vm);
-  const namespace = getNamespace(vm);
-  const uid = getUID(vm);
-  const lookupID = getBasicID(vm);
-
+  const name = getName(obj);
+  const namespace = getNamespace(obj);
+  const uid = getUID(obj);
+  const lookupID = getBasicID(obj);
   const migration = migrationLookup[lookupID];
-  const vmi = vmiLookup[lookupID];
-  const vmStatus = getVMStatus({ vm, vmi, pods, migrations });
+
+  let vm = null;
```

Comment:
is the `= null` necessary in these vars?

Reply:
Thanks :heart: fixed